servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -47,7 +47,9 @@
  * Default implementation of {@link HttpLoadBalancerFactory}.
  *
  * @param <ResolvedAddress> The type of address after resolution.
+ * @deprecated use {@link io.servicetalk.http.api.DefaultHttpLoadBalancerFactory} instead.
  */
+@Deprecated // FIXME: 0.43 - remove deprecated class
```

Comment:
Consider also clarifying in `Builder.fromDefaults()` javadoc how we expect users to migrate